### ISSUE TITLE
ECMDERR Failed to execute "git ls-remote --tags --heads git://localhost:6789/highcharts"

### ISSUE THREAD

**Issue by glispajon** [Hacklone/private-bower#152]
Like issue #102 I'm getting errors when trying to install non-private packages. It seems rather random which one fails but on my client machine I get:
```
bower highcharts#~4.1.3                 ECMDERR Failed to execute "git ls-remote --tags --heads git://localhost:6789/highcharts", exit code of #128 fatal: unable to access 'https://localhost:6789/highcharts/': Failed to connect to localhost port 6789: Connection refused

Additional error details:
fatal: unable to access 'https://localhost:6789/highcharts/': Failed to connect to localhost port 6789: Connection refused
```
So it seems like my client is trying to access localhost:6789, which doesn't run the git daemon. Running "git ls-remote --tags --heads git://localhost:6789/highcharts" on the server with private-bower works fine. 

Any idea why I'd get this problem? It seems random which dependency it is that causes the problem but it's always saying localhost  (on my client machine). The log from private-bower doesn't report any problems.

I don't have any problem with a firewall or git/https, but I don't run a local git daemon. Do I have to do that?

When I try accessing the private-bower server directly on ' /packages/highcharts' does everything seems fine:
```
{"name":"highcharts","url":"git://github.com/highslide-software/highcharts-release.git","hits":106077}
```

**Comment by Hacklone** [comment]
Hey,

If you call this url: ```http://you-private-bower-server:5678/packages/highcharts``` it gives you: ```{"name":"highcharts","url":"git://github.com/highslide-software/highcharts-release.git","hits":106077}``` 
That means that you've registered that package for the github url -> your client should call this same api, so it wouldn't try to download it from the localhost.

If your client still tries to download it from the localhost I presume your ```.bowerrc``` file's registry entry points to a registry which provides you with a localhost url for the package and not to your private-bower server (or another instance of it :))

**Comment by glispajon** [comment]
Hi,

You were completely right. While trying out private-bower I installed it locally as well to try it out before setting up the server. For some reason bower got confused for some packages (I did "bower cache clean" repeatedly) by the local server and for some packages (seemed random) it tried to get the packages from localhost. I stopped the local server this morning and now everything installed fine.

Thanks for the quick reply and sorry for wasting your time!

/Jon
